features/raadpleeg-verblijfplaats-met-peildatum/gba/test/opschorting-bijhouding-gba.feature

```diff
@@ -0,0 +1,172 @@
+
+    Abstract Scenario: historie wordt gevraagd van persoon waarvoor de bijhouding is opgeschort met peildatum <omschrijving> datum opschorting (overlijden)
+      Gegeven de persoon met burgerservicenummer '000000012' is ingeschreven op adres 'A1' met de volgende gegevens
+      | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+      | 0800                              | 20160801                           |
+      En de persoon heeft de volgende 'inschrijving' gegevens
+      | naam                                 | waarde   |
+      | reden opschorting bijhouding (67.20) | O        |
+      | datum opschorting bijhouding (67.10) | 20220829 |
+      Als gba verblijfplaatshistorie wordt gezocht met de volgende parameters
+      | naam                | waarde                |
+      | type                | RaadpleegMetPeildatum |
+      | burgerservicenummer | 000000012             |
+      | peildatum           | <peildatum>           |
+      Dan heeft de response verblijfplaatsen met de volgende gegevens
+      | gemeenteVanInschrijving.code | gemeenteVanInschrijving.omschrijving | datumAanvangAdreshouding | straat           |
+      | 0800                         | Hoogeloon, Hapert en Casteren        | 20160801                 | Korte straatnaam |
+      En heeft de response de volgende gegevens
+      | naam                                     | waarde     |
+      | opschortingBijhouding.reden.code         | O          |
+      | opschortingBijhouding.reden.omschrijving | overlijden |
+      | opschortingBijhouding.datum              | 20220829   |
+
+      Voorbeelden:
+      | omschrijving | peildatum  | datum tot  |
+      | voor         | 2020-01-01 | 2021-01-01 |
+      | gelijk aan   | 2022-08-29 | 2023-01-01 |
+      | na           | 2023-01-01 | 2023-07-01 |
+
+    Abstract Scenario: historie wordt gevraagd van persoon waarvoor de bijhouding is opgeschort voor peildatum waarop geen verblijfplaats wordt gevonden
+      Gegeven de persoon met burgerservicenummer '000000012' is ingeschreven op adres 'A1' met de volgende gegevens
+      | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+      | 0800                              | 20220801                           |
+      En de persoon heeft de volgende 'inschrijving' gegevens
+      | naam                                 | waarde                         |
+      | reden opschorting bijhouding (67.20) | <reden opschorting bijhouding> |
+      | datum opschorting bijhouding (67.10) | 20220829                       |
+      Als gba verblijfplaatshistorie wordt gezocht met de volgende parameters
+      | naam                | waarde                |
+      | type                | RaadpleegMetPeildatum |
+      | burgerservicenummer | 000000012             |
+      | peildatum           | 2022-07-01            |
+      Dan heeft de response geen 'verblijfplaatsen' gegevens
```

Comment:
Is dit hetzelfde als `Dan heeft de response 0 verblijfplaatsen`? Dus lege array of geen verblijfplaatsen veld?

Reply:
zou moeten kunnen. toen ik dit maakte was er alleen nog een oude versie van stepdefs die niet goed werkte met deze stap en daarna nog een Dan stap met andere data buiten de verblijfplaatsen (in dit geval opschorting).
Als dat nu wel werkt, verander ik de stap.

Reply:
Welke gaat het nu worden?